src/System.Runtime.CompilerServices.Unsafe/pkg/System.Runtime.CompilerServices.Unsafe.pkgproj

```diff
@@ -7,6 +7,10 @@
       <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
+
+    <!-- this package is part of the implementation closure of NETStandard.Library
+         therefore it cannot reference NETStandard.Library -->
+    <SuppressMetaPackage Include="NETStandard.Library" />
```

Comment:
Are there other packages this should be added to, as well?

Reply:
> Interestingly, System.Security.Cryptography.Cng does reference System.Memory, but doesn't trigger the circular dependency:

That's because CNG isn't in the package closure of NETStandard.Library.  The problem comes about when something in the closure of NETStandard.Library starts referencing NETStandard.Library.  For everything subsumed by NETStandard.Library this isn't an issue since we no longer build packages for them.  There were, however, a handful of packages which were previously in the implementation closure of NETStandard.Library, but not added to the NETStandard2.0 surface area (not subsumed):

- System.Buffers
- System.Diagnostics.DiagnosticSource
- System.Threading.Tasks.Extensions
- System.Reflection.Emit
- System.Reflection.Emit.ILGeneration
- System.Reflection.Emit.Lightweight
- System.Reflection.TypeExtensions
- System.Security.Cryptography.OpenSsl

For any of these we have to make sure that newer versions of the packages don't bring in a transitive reference to NETStandard.Library, otherwise it will create a cycle.  The one that did introduce this was System.Reflection.TypeExtensions > System.Runtime.CompilerServices.Unsafe > NETStandard.Library.

Reply:
> System.Reflection.TypeExtensions > System.Runtime.CompilerServices.Unsafe > NETStandard.Library

Shouldn't it be `System.Threading.Tasks.Extensions`?

Reply:
Right you are.  Corrected.